yodeploy/hooks/nginx.py

```diff
@@ -0,0 +1,56 @@
+"""Application hooks for apps that use nginx."""
+
+import logging
+import os
+import subprocess
+import sys
+
+from yodeploy.hooks.configurator import ConfiguratedApp
+from yodeploy.util import touch
+
+log = logging.getLogger(__name__)
+
+
+class Nginx(object):
```

Comment:
I'm not sure it worth a separate class. Maybe just add `_reload_nginx` method to the hook.

Reply:
I want to be consistent with existing code (Apache hooks for example)

Reply:
And it is easier to plug mock for testing